docs/QuickStart-Serialization.md

```diff
@@ -0,0 +1,66 @@
+---
+id: quickstart-api-serialization
+title: Serialization
+layout: docs
+category: Quick Start
+next: advanced-topics-entities
+permalink: docs/quickstart-serialization.html
+---
+
+# Serialization
+
+Because a text editor doesn't exist in a vacuum and it's important to save 
+contents for storage or transmission, you will want to be able to convert 
+`ContentState` into a plain JS object, and vice versa. For storing it in a database,
+it may be easiest to store it in a string field as json.
```

Comment:
I suggest also linking to the "Data Conversion" doc to provide direct API reference: https://facebook.github.io/draft-js/docs/api-reference-data-conversion.html#content